CryptoPrices/Sources/Home/Sources/Home/TrendingCoinsSectionView/TrendingCoinItem.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  TrendingCoinItem.swift
+//  Home
+//
+//  Created by Minh Pham on 29/12/2022.
+//
+
+import Entities
+import Foundation
+
+public struct TrendingCoinItem: Identifiable, Equatable {
+    
+    public let id: String
+    public let symbol: String
+    public let name: String
+    public let iconUrl: URL
+    public let priceChangePercentage: Double
+    public let isPriceUp: Bool
+    
+    init(
+        id: String,
```

Comment:
Yeah, we can omit the `isPriceUp` from the initializer since it's derived from `priceChangePercentage`, not meant to directly get the value from the outside:
```swift
public init(  👈 please help to make this public
  id: String,
  symbol: String,
  name: String,
  iconUrl: URL,
  priceChangePercentage: Double
) {
  self.id = id
  self.symbol = symbol.uppercased()
  self.name = name
  self.iconUrl = iconUrl
  self.priceChangePercentage = priceChangePercentage
  self.isPriceUp = priceChangePercentage > 0.0
}
```

@nkhanh44 About removing this initiazier completely, I think this is a `public` type, so I think it deserves an (_normal_) `public` initializer. If we remove this or we we leave it but without `public`, other modules only can init an instance from `Coin`. (Sometimes the other module may just want init one with individual paramaters). That's because the memberwise initializer (synthesized by the compiler) is `internal` by default. We can think `init(coin: Coin)` is just a secondary (or `convenience` for `class`) initialiazer. This is the `primary` initializer, like an official one 🙏 

@minhnimble Please help me update the same with `MyCoinItem` too 🙏

Reply:
> About removing this initializer completely, I think this is a public type, so I think it deserves an (normal) public initializer. If we remove this or we leave it but without public, other modules only can init an instance from Coin. (Sometimes the other module may just want to init one with individual parameters). That's because the memberwise initializer (synthesized by the compiler) is internal by default. We can think init(coin: Coin) is just a secondary (or convenience for class) initializer. This is the primary initializer, like an official one 🙏

-> @Thieurom actually there is not an immediate need for the `primary` initializer to be honest, and maybe we can add it back when there is an actual need for a clearer picture. Thus, removing it for now is not a bad call at all, since we should also think about implementing the minimal amount of code to get the job done, as per some of [the great benefits](https://medium.com/helpshift-engineering/being-a-minimalist-when-it-comes-to-code-ffcc807886ea) this provides. What do you think? 🤔

> Please help me update the same with MyCoinItem too 🙏

-> Sure, will do 👍

Reply:
@minhnimble Yeah, it's fair not bothering with this right now 🙏